export/tests/test_export_tool.py

```diff
@@ -53,3 +58,94 @@ def test_gen_filename(self):
+
+    def setUp(self):
+        self.client.login(username="super", password="super007")
+
+    def test_export_xml(self):
+        response = self.client.get(path="/admin/auth/user/")
+        self.assertEquals(response.status_code, 200)
+        response = self.client.get(self.export_url)
+        self.assertEquals(response.status_code, 200)
+        export_form_data_asc = {
+            "export_format": "xml",
+            "export_order_by": "username",
+            "export_order_direction": "asc"
+        }
+        export_form_data_dsc = {
+            "export_format": "xml",
+            "export_order_by": "username",
+            "export_order_direction": "dsc"
+        }
+        response_asc = self.client.post(
+            path=self.export_url,
+            data=export_form_data_asc,
+            follow=True
+        )
+        response_dsc = self.client.post(
+            path=self.export_url,
+            data=export_form_data_dsc,
+            follow=True
+        )
+        content_type = response_asc["content-type"]
+        self.assertEquals(content_type, "application/xml")
+        self.assertNotEqual(response_asc, response_dsc)
+
+    def test_export_json(self):
+        export_form_data_jsn = {
+            "export_format": "json",
+            "export_order_by": "username",
+            "export_order_direction": "asc"
+        }
+        response = self.client.post(
+            path=self.export_url,
+            data=export_form_data_jsn,
+            follow=True
+        )
+        content_type = response["content-type"]
+        self.assertEquals(content_type, "application/json")
+        json_content = json.loads(response.content.decode("utf-8"))
+        objects = User.objects.count()
+        self.assertEquals(type(json_content[0]), dict)
+        self.assertEquals(json_content[0]["model"], "auth.user")
+        self.assertEquals(objects, len(json_content))
+
+    def test_export_csv(self):
+        User.objects.create_user(
+            username="FALSE",
+            email="someuser@user.com",
+            password="another007",
+            first_name="[test user]",
+            last_name="NULL"
+        )
+        export_form_data_csv = {
+            "export_format": "csv",
+            "export_order_by": "username",
+            "export_order_direction": "asc"
+        }
+        response = self.client.post(
+            path=self.export_url,
+            data=export_form_data_csv,
+            follow=True
+        )
+        content_type = response["content-type"]
+        self.assertEquals(content_type, "text/csv")
+        # Check csv serialization rules as defined in csv_serializer.py
+        self.assertContains(response, "TRUE")
+        self.assertContains(response, "FALSE")
+        self.assertContains(response, "NULL")
+        self.assertContains(response, "'[test user]'")
+        self.assertContains(response, "'FALSE'")
+        self.assertContains(response, "'NULL'")
```

Comment:
If the response contains "'FALSE'" it will contain "FALSE". This needs refactoring to something close to the full line to make sure we are testing for the correct text in the correct place.